go/exocom/src/security/securty.go

```diff
@@ -0,0 +1,57 @@
+package security
+
+import (
+	"fmt"
+
+	"github.com/Originate/exocom/go/structs"
+	uuid "github.com/satori/go.uuid"
+)
+
+// Manager is the struct that handles incoming security
+// messages
+type Manager struct {
+	authTable map[string]structs.Message
+}
+
+// Result is the result of a secuirty manager parsing a
+// given message
+type Result struct {
+	MessageToSend  structs.Message
+	WarningMessage string
+}
+
+// NewSecurityManager returns a new security manager
+func NewSecurityManager() *Manager {
+	result := &Manager{
+		authTable: map[string]structs.Message{},
+	}
+	return result
+}
+
+// ReceiveMessage handles incoming security messages
+func (m *Manager) ReceiveMessage(message structs.Message) *Result {
```

Comment:
This method is now nicely unit testable. Thoughts on adding some unit tests for it? Mainly makes it easier to develop it (especially for part 2) as the unit tests will run much faster than the feature tests

Reply:
You bet